packages/redux/src/users/authentication/__tests__/selectors.test.ts

```diff
@@ -134,4 +140,28 @@ describe('authentication redux selectors', () => {
       expect(selectors.isAuthenticated(mockState)).toBe(true);
     });
   });
+
+  describe('areExternalLoginsLoading()', () => {
```

Comment:
Missing test for `getExternalLogins` selector.

Reply:
Doesn't the `describe('sub-areas with result', () => {}` tests for the result?
I've added the `ExternalLogins` sub-area to that test.

Reply:
You are right, my mistake.